initializr-service/Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM openjdk:8-jre-alpine
-MAINTAINER yaweiw@microsoft.com
```

Comment:
By the way, same maintainer info in another Dockerfile:

https://github.com/yaweiw/azure-spring-cloud-playground/blob/master/initializr-generator/src/main/resources/templates/docker/Dockerfile#L2

Reply:
I will cover that.